### ISSUE TITLE
verb-short forms in Hebrew

### ISSUE THREAD

**Issue by demai** [citation-style-language/locales#193]
In locales-he-IL.xml verb short forms are still in English. Is this intentional or should we localize those terms as well?

**Comment by rmzelle** [comment]
Whenever we start a locale file for a new language, or add new CSL terms (e.g. to accompany a new version of CSL), we prepopulate the files with English translations. So these terms just never have been translated, and should be localized as well.

**Comment by demai** [comment]
Ok, I'll look if I can come up with some terms here. Should I just file a pull request then?

**Comment by rmzelle** [comment]
Yes, pull requests are the easiest way to propose any changes to the locale files.
